huma_signals/adapters/lending_pools/adapter.py

```diff
@@ -35,6 +38,9 @@ class LendingPoolSignals(models.HumaBaseModel):
     token_decimal: int = pydantic.Field(
         ..., description="Decimal of the token used by the pool"
     )
+    nft_contract_address: str = pydantic.Field(
```

Comment:
Not sure if this will ever be useful even if we bring back EA NFT, because it'll mostly be used for routing underwriting requests to the appropriate NFTs.

Reply:
Ok removed